docs/snippets/all/concepts/recording_properties.cpp

```diff
@@ -2,15 +2,59 @@
 
 #include <rerun.hpp>
 
-int main() {
+#include <arrow/array/builder_binary.h>
+#include <arrow/array/builder_primitive.h>
+
+arrow::Status run_main() {
     const auto rec = rerun::RecordingStream("rerun_example_recording_properties");
     rec.spawn().exit_on_failure();
 
-    auto properties = rerun::archetypes::RecordingProperties().with_start_time(0).with_name(
-        "My recording (initial)"
+    // Overwrites the name from above.
+    rec.send_recording_name("My recording");
+
+    // Start time is set automatically, but we can overwrite it at any time.
+    rec.send_recording_start_time_nanos(1742539110661000000);
+
+    auto points = rerun::Points3D({{1.0, 0.1, 1.0}});
+    rec.send_property("camera_left", points);
+
+    std::shared_ptr<arrow::Array> arrow_array;
```

Comment:
```suggestion
    // Adds another property, this time with user-defined data.
    {
          std::shared_ptr<arrow::Array> arrow_array;
          // ...
```

I think this helps guiding through the example a little bit just like you did in Rust & Python :)